README.md

```diff
@@ -110,6 +110,13 @@ To enable/disable the autostart service perform the following steps:
   $ chkconfig --del autostartliota
 ```
 
+### On systemd based systems:
+
+```bash
+  $ systemctl enable liota.socket
```

Comment:
@tompscanlan 

It'll be very helpful if you can provide some information about this change.